pyqtgraph/functions.py

```diff
@@ -2951,6 +2953,34 @@ def invertQTransform(tr):
         return _pinv_fallback(tr)
     
 
+def turnInfToSysMax(val):
+    """Turns infinite values into sys.float_info.max and -infinite values into -sys.float_info.max.
+       Used for QPointF and QTransform or normal float values.
+    """
+    if isinstance(val, QtCore.QPointF):
+        return QtCore.QPointF(turnInfToSysMax(val.x()), turnInfToSysMax(val.y()))
+    if isinstance(val, QtGui.QTransform):
+        return QtGui.QTransform(turnInfToSysMax(val.m11()), turnInfToSysMax(val.m12()), turnInfToSysMax(val.m13()),
+                                turnInfToSysMax(val.m21()), turnInfToSysMax(val.m22()), turnInfToSysMax(val.m23()),
+                                turnInfToSysMax(val.m31()), turnInfToSysMax(val.m32()), turnInfToSysMax(val.m33()))
+    if val == math.inf:
+        return sys.float_info.max
+    if val == -math.inf:
+        return -sys.float_info.max
+    return val
+
+
+def np_invert_qtransform(tr):
```

Comment:
Why use this instead of https://github.com/pyqtgraph/pyqtgraph/blob/master/pyqtgraph/functions.py#L2927-L2933

the pseudo-inversion would handle cases where even if the determinant is 0?